application/app.py

```diff
@@ -268,7 +269,7 @@ def play_card():
     from_discard_pile = false_true(request.args.get('from_discard_pile'))
     trade = false_true(request.args.get('trade'))
 
-    if process_card(card, player, discarded, for_wonder, from_discard_pile=from_discard_pile, trade=trade):
+    if process_card(card, player, discarded, for_wonder, from_discard_pile=from_discard_pile, tr=trade):
```

Comment:
Hmm changing trade to tr is a bit implicit. I'd say should_trade is a bit more obvious and easier to read

Reply:
Okay, will change that on the next PR I push (today).